infra/charts/turing/templates/virtualservice-mlp.yaml

```diff
@@ -85,6 +97,22 @@ spec:
             host: {{ template "turing.merlin.name" . }}
             port:
               number: 8080
+    - match:
+        - uri:
+            prefix: "/xp"
+      route:
+        - destination:
+            host: {{ template "turing.xpManagement.name" . }}
+            port:
+              number: 8080
+    - match:
+        - uri:
+            prefix: "/xp-treatment"
```

Comment:
Should we make it `/api/xp-treatment` for consistency with other APIs?

Reply:
Updated this!